zerver/lib/import_realm.py

```diff
@@ -1425,6 +1426,14 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
         update_model_ids(AlertWord, data, "alertword")
         bulk_import_model(data, AlertWord)
 
+    if "zerver_savedsnippet" in data:
+        re_map_foreign_keys(
+            data, "zerver_savedsnippet", "user_profile", related_table="user_profile"
+        )
+        re_map_foreign_keys(data, "zerver_savedsnippet", "realm", related_table="realm")
+        update_model_ids(SavedSnippet, data, "savedsnippet")
+        bulk_import_model(data, SavedSnippet)
```

Comment:
How have you tested this code? Usually we extend the data in the existing `test_import_export` tests to creates one object and verify it survives the data export/import process.

Reply:
I tried adding it to `test_import_export` in `SingleUserExportTest` and creating a `checker` for `zerver_savedsnippet` but I am getting an error that there is no `zerver_savedsnippet` key in `user.json`.
```sh
Traceback (most recent call last):
  File "/home/vector/zulip/zerver/tests/test_import_export.py", line 2289, in test_user_data
    f(user[table_name])
KeyError: 'zerver_savedsnippet'
```

Reply:
OK, that probably means this code has a bug.